src/packet/public_key_encrypted_session_key.js

```diff
@@ -106,7 +106,7 @@ class PublicKeyEncryptedSessionKeyPacket {
     ]);
     const algo = enums.write(enums.publicKey, this.publicKeyAlgorithm);
     this.encrypted = await crypto.publicKeyEncrypt(
-      algo, key.publicParams, data, key.getFingerprintBytes());
+      algo, key.publicParams, key.privateParams, data, key.getFingerprintBytes());
```

Comment:
Instead of always passing the private params here, I would rather conditionally pass the private params instead of the public params if it's a symmetric algorithm. I know it's a little bit hacky, but that way there's fewer lines of code changed.
And similarly for `signature.verify` below.

Reply:
You need both, since the public params hold the symmetric cipher.

Reply:
Ah yeah, alrighty.